requirements.txt

```diff
@@ -1,4 +1,4 @@
+future
 httplib2
 lxml
-ordereddict
```

Comment:
Drop the try-except ordereddict imports?

Reply:
It was to keep `py2.6` support. dropping with this change!